modules/dcache/src/main/java/org/dcache/pool/migration/NotFilter.java

```diff
@@ -1,23 +1,24 @@
 package org.dcache.pool.migration;
 
+import java.util.function.Predicate;
 import org.dcache.pool.repository.CacheEntry;
 
 /**
  * Repository entry filter wrapping another filter. It accepts all
  * entries not accepted by the other filter.
  */
-public class NotFilter implements CacheEntryFilter
+public class NotFilter implements Predicate<CacheEntry>
```

Comment:
As you use standard Predicate, then there are no reasons to keep NotFilter(which is not used anyway)

Reply:
I cleaned up the classes in the second commit of this pull request and removed NotStickyOwnerFilter.
I indeed overlooked NotFilter.

Reply:
Fixed in 1fcf3a53b955616eb7c06c1e7a5b880880dd96f8